cli/src/main/scala/com/advancedtelematic/tuf/cli/repo/TufRepo.scala

```diff
@@ -262,10 +261,12 @@ class TufRepo(val name: RepoName, val repoPath: Path)(implicit ec: ExecutionCont
     signRole[TargetsRole](version, targetsKeys)
 
   private val VersionPath = ^.version
+  private val ExpiresPath = ^.expires
   import cats.implicits._
 
   private def signRole[T : Decoder : Encoder](version: Option[Int], keys: Seq[KeyName])
-                                             (implicit ev: TufRole[T], versionL: VersionPath.Lens[T, Int]): Try[Path] = {
+                                             (implicit ev: TufRole[T], versionL: VersionPath.Lens[T, Int],
+                                              expiresL: ExpiresPath.Lens[T, Instant]): Try[Path] = {
```

Comment:
this is not needed.

Reply:
Weird. The issue came from a customer.

Reply:
I meant the expiresL implicit is not needed but I was wrong.